Campus-iOS/GradesComponent/ViewModel/GradesViewModel.swift

```diff
@@ -8,105 +8,151 @@
 import Foundation
 import SwiftUI
 
-protocol GradesViewModelProtocol: ObservableObject {
```

Comment:
Not sure if this is a good idea. Technically, this would enable easy creation of Mock View Models that can be used for Testing